controller/internal/processmon/processmon.go

```diff
@@ -31,7 +31,7 @@ const (
 	netNSPath                   = "/var/run/netns/"
 	processMonitorCacheName     = "ProcessMonitorCache"
 	remoteEnforcerBuildName     = "remoteenforcerd"
-	remoteEnforcerTempBuildPath = "/var/run/aporeto/tmp/bin/"
+	remoteEnforcerTempBuildPath = "/opt/aporeto/tmp/bin/"
```

Comment:
I just realized that this is going to increase our startup times by several milliseconds. /var/run is memfs .. is /opt/aporeto/tmp memfs ?

Reply:
nope i will fix this with a later pull request to make these independent binaries which will be copied to memfs when the enforcer starts